integritybackend/claim.py

```diff
@@ -262,6 +305,44 @@ def _get_meta_lat_lon(self, meta):
 
         return (float(lat), float(lon))
 
+    # def _make_photo_meta_data(self, jwt_payload, meta):
```

Comment:
Claim generation file will be cleaned up so func work for both Starling Capture and Proofmode. Right now legacy Starling Capture logic is commented out, as they need to stop relying on `jwt_payload` and move to `-meta-content.json` to construct claim.